coopterrones/color-palette
# Feature/card controller

#### What's this PR do?
- Displays the Colors passed down through props to ColorCards
- Displays color hexCodes below color window
- Viewable in browser
- Conditional render bug fixed -- cards mount before color state is passed down 

#### Where should the reviewer start?
- Line 22 `App.js` -- hexCode converter 
- Line 40 `App.js` -- conditional render of components 
- Line 1 `CardController.js` -- Renders ColorCard components and passes state down 
- Line 1 `ColorCard.js` -- Color cards created 
- CSS added for each component initially to display 

#### How should this be manually tested?
- Test for load of random colors on refresh of page -- check screenshot for reference 
- Hexcodes should display bellow colors 

#### Screenshots (if appropriate):

![Screen Shot 2021-01-14 at 5 33 40 PM](https://user-images.githubusercontent.com/67164959/104666701-bc0ac080-5691-11eb-9608-f68ce2aac2fe.png)
